projects/fsastorefrontlib/src/core/checkout/store/effects/product.effect.ts

```diff
@@ -16,18 +20,31 @@ export class ProductEffect {
       return this.productPricingConnector
         .getCalculatedProductData(payload.productCode, payload.pricingData)
         .pipe(
+          timeout(30000),
```

Comment:
As timeout is browser dependant, it's agreed with PO that appropriate time for waiting for the response of this product calculation request is 30 seconds.